### PR TITLE
Upload DASH PDMs using WebMTP (UPLOAD-349)

### PR DESCRIPTION
**Pull request by gniezen**
Addresses [UPLOAD-349].

[UPLOAD-349]: https://tidepool.atlassian.net/browse/UPLOAD-349

### CODE REVIEW

File: lib/drivers/insulet/insuletDriver.js
Comment (pazaan) [inline lib/drivers/insulet/insuletDriver.js:1761]:
```suggestion
          const webmtp = require('webmtp');
          const mtp = new webmtp(deviceInfo.vendorId, deviceInfo.productId);
```

Perhaps a little pedantic, but I always get a bit nervous around variables that only vary by case. It's not bad in this case, because it's only 2 lines. Optional suggestion 😄

**Review by pazaan — APPROVED** [review]
One optionally annotated nit. LGTM! :shipit:

File: lib/drivers/insulet/insuletDriver.js
Comment (gniezen) [reply lib/drivers/insulet/insuletDriver.js]: Added it, and then changed it to `WebMTP`, as our linting rules require constructors to start with a capital letter.

**Review by pazaan — APPROVED** [review]
LGTM

**Comment by gniezen** [comment]
Sorry, need another approval after resolving a merge conflict.

**Review by pazaan — APPROVED** [review]
LGTM

**Comment by gniezen** [comment]
Passed QA, so removing test suffix for final approval.

**Review by pazaan — APPROVED** [review]
LGTM 📦

**Pull request merged**
